### PR TITLE
Fix hard coded db connection string to allow connection to non-root users

### LINKED ISSUE #168: Non-root database user and password
We need to fix the code to be able to use a database user which is not necessary root. Check the hard coded connection string that we have now instead: https://github.com/Clinical-Genomics/chanjo2/blob/5c85efecd1afa92056cbbfce5fa39697dfaa55eb/src/chanjo2/dbutil.py#L31

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Fix #168 
- This fix will allow connecting to real database on cg-vm1 and cg-prod-services

### How to test:
- Test the changes by running the dockerized version of the repo connected to a docker database: `docker-compose -f docker-compose-mysql.yml --env-file template.env up`

### Expected outcome:
- [x] The server should be launched and serving pages without problems

### Review:
- [x] Code approved by HS
- [x] Tests executed by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
Tested locally connected with a dockerized MariaDB container: 🆗 

image


image

File: docker-compose-mysql.yml [hunk 1]
```diff
@@ -7,11 +7,12 @@ services:
       dockerfile: Dockerfile-db # MariaDB with an init.sql file to initialize the database
     restart: 'always'
     environment:
-      - MYSQL_USER=${DB_USER_NAME}
-      - MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD}
-      - MYSQL_DATABASE_NAME=${MYSQL_DATABASE_NAME}
+      - MYSQL_USER=${MYSQL_USER}
+      - MYSQL_PASSWORD=${MYSQL_PASSWORD}
+      - MYSQL_DATABASE=${MYSQL_DATABASE_NAME}
+      - MARIADB_RANDOM_ROOT_PASSWORD=T
```
Comment (henrikstranneheim) [inline docker-compose-mysql.yml:13]: How does this work?

**Review by henrikstranneheim — COMMENTED** [review]
Nice 👍

File: docker-compose-mysql.yml (hunk 1 above)
Comment (northwestwitch) [reply docker-compose-mysql.yml:13]: MariaDB doesn't start if you don't provide a root password (as MYSQL_ROOT_PASSWORD or MARIADB_ROOT_PASSWORD), so you either provide one or you let the container create one for you. It's not used at all in chanjo2 where the user and password are MYSQL_USER and MYSQL_PASSWORD

**Pull request merged**